python/turbine_models/tests/conftest.py

```diff
@@ -0,0 +1,18 @@
+def pytest_addoption(parser):
+    parser.addoption("--all", action="store_true", help="run all combinations")
+
+
+def pytest_generate_tests(metafunc):
+    if "quantization" in metafunc.fixturenames:
+        if metafunc.config.getoption("all"):
+            quantizations = ["int4", None]
```

Comment:
no int8?

Reply:
Removed. Will add in once it's time to do bigger CI